docs/docs-content/vm-management/create-manage-vm/advanced-topics/migrate-vm-kubevirt.md

```diff
@@ -80,6 +80,64 @@ from VMware vSphere to Palette VMO.
   - The Palette CLI must have access to both the VMO cluster and the machines to be migrated.
 - The kubectl command-line tool should also be installed. Refer to the
   [kubectl installation](https://kubernetes.io/docs/tasks/tools/install-kubectl/) guide to learn more.
+- We recommend providing a VMware Virtual Disk Development Kit (VDDK) image for the migration. This will significantly
+  speed up the migration.
+
+  - The VDDK image must be built and uploaded to your image registry before starting the migration. Refer to the
+    [Red Hat Documentation](https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.6/html/installing_and_using_the_migration_toolkit_for_virtualization/prerequisites_mtv#creating-vddk-image_mtv)
+    for guidance.
+  - The host with the Palette CLI setup for migration must have access to your image registry.
+  - If you are using a private image registry, you must create a Secret to be used for the migration. The Secret must be
+    in the form of a YAML file and the `metadata.name` value must be `vddk-image-pull-secret`.
+
+    <!--prettier-ignore-->
+    <details>
+    <summary> Example Secret Creation </summary>
+
+    A Secret can be created by issuing the following command.
+
+    ```shell
+    kubectl create secret docker-registry vddk-image-pull-secret \
+    --docker-server=myRegistryServer \
+    --docker-username=myUsername \
+    --docker-password=myPassword \
+    --docker-email=myEmail \
+    -o yaml > image-pull-secret.yaml
+    ```
+
+    This creates the `image-pull-secret.yaml` file in your working directory.
+
+    ```yaml title="Example contents of image-pull-secret.yaml"
```

Comment:
I'm not sure we've ever used titles on our snippets 🤔 Also please add `hideClipboard` on any of these non-copy snippets.

Reply:
They are useful (snippet titles that is), but I will remove for now to be consistent.

Reply:
```suggestion
    ```yaml hideClipboard
```